projects/igniteui-angular/src/lib/progressbar/linearbar.component.spec.ts

```diff
@@ -475,6 +475,23 @@ describe('IgLinearBar', () => {
             fix.detectChanges();
             expect(bar.classList.contains(INDETERMINATE_CLASS)).toEqual(true);
         });
+
+        it('Check if current value in % changes dynamically when max is increased', fakeAsync(() => {
+            const fix = TestBed.createComponent(InitLinearProgressBarComponent);
+            fix.detectChanges();
+
+            const bar = fix.componentInstance.linearBar;
+            const maxVal = 3.25;
+            const val =  3.55;
+
+            bar.max = maxVal;
+            bar.value = val;
+            const valueInPercent = Common.calcPercentage(bar.value, bar.max);
```

Comment:
You can simulate max value increase and double-check the valueInPercent prop to make the test more valid.
### Example:
**first scenario:**
1. set max to 100
2. set value to 50
3. check valueInPercent to equal 50%

**second scenario:**
1. set max to 200
2. check valueInPercent to equal 25%